pkg/apic/provisioning/accessrequestdefinitionbuilder.go

```diff
@@ -35,10 +37,20 @@ func NewAccessRequestBuilder(registerFunc RegisterAccessRequestDefinition) Acces
 
 // SetName - set the name of the access request
 func (a *accessRequestDef) SetName(name string) AccessRequestBuilder {
+	if a.title == "" {
```

Comment:
Do we need this check/assignment here as the Register method is already doing this anyways?